setup.py

```diff
@@ -119,7 +119,6 @@ def run(self):
               'Environment :: Plugins',
               'Environment :: MacOS X',
               'Environment :: Win32 (MS Windows)',
-              'Environment :: X11 Applications :: Gnome',
```

Comment:
This change doesn't help the PR. If you think this is an valid change start a newcomer issue maybe?